docs/topics/modules/closing_workitem.adoc

```diff
@@ -3,11 +3,11 @@
 
 After implementing and reviewing your code changes, update the relevant Work Item to track the work done.
 
-. Click the btn:[Plan] tab and then btn:[Board] in the options to view the Planner Board view.
-. Drag the Work Item to the `Resolved` column.
+. Click the *Plan* tab and then *Board* to see the Planner Board view.
+. Drag the Work Item to the *Resolved* state.
 +
 image::resolved_wi.png[Resolved Work Item]
 
-Closing or resolving a Work Item communicated to the team that the task being tracked is complete.
+Resolving a Work Item communicates to the team that the task being tracked is complete. You can drag it to the *Closed* state, after it is reviewed and closed.
```

Comment:
This sounds strange: "You can drag it to the *Closed* state, after it is reviewed and closed." -- So, you close it (how?), and then you can drag it to 'Closed'?

I would expect that you close it **by** dragging it to 'Closed'.